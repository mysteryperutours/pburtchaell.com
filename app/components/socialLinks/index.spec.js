import React from 'react';
import SocialLinks from './index';

describe(SocialLinks.name, () => {
  it('should render component', () => {
    expect(shallow(
      <SocialLinks />
    )).toMatchSnapshot();
  });
});
