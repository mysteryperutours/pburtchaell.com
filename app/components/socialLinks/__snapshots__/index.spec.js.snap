exports[`SocialLinks should render component 1`] = `
<ul
  className="social-links-list">
  <li
    className="social-links-list-item">
    <a
      href="https://facebook.com/pburtchaell"
      title="Patrick Burtchaell on Facebook">
      <Icon
        type="facebook" />
      Facebook
    </a>
  </li>
  <li
    className="about-links-list-item">
    <a
      href="https://twitter.com/pburtchaell"
      title="Patrick Burtchaell on Twitter">
      <Icon
        type="twitter" />
      Twitter
    </a>
  </li>
  <li
    className="about-links-list-item">
    <a
      href="https://github.com/pburtchaell"
      title="Patrick Burtchaell on GitHub">
      <Icon
        type="github" />
      GitHub
    </a>
  </li>
  <li
    className="about-links-list-item">
    <a
      href="https://dribbble.com/pburtchaell"
      title="Patrick Burtchaell on Dribbble">
      <Icon
        type="dribbble" />
      Dribbble
    </a>
  </li>
  <li
    className="about-links-list-item">
    <a
      href="mailto:patrick@pburtchaell.com"
      title="Patrick Burtchaell on email">
      patrick@pburtchaell.com
    </a>
  </li>
</ul>
`;
